HyperVExtension/src/HyperVExtension/HyperVExtension.csproj

```diff
@@ -40,7 +40,7 @@
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
-    <PackageReference Include="Microsoft.Windows.DevHome.SDK" Version="0.300.443" />
+    <PackageReference Include="Microsoft.Windows.DevHome.SDK" Version="0.400.460" />

```

Comment:
There is another reference to Microsoft.Windows.DevHome.SDK in CoreWidgetProvider.csproj

Reply:
Yep I know it's already in this PR